pom.xml

```diff
@@ -49,24 +49,32 @@
         <revision>0.3.3</revision>
         <changelist>-SNAPSHOT</changelist>
         <java.level>8</java.level>
-        <jcasc.version>1.36</jcasc.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-${jenkins.version}.x</artifactId>
```

Comment:
You can't use the `${jenkins.version}` without removing the last digit 😅

Reply:
When I looked in the Plugin POM, that property is defined only as `<jenkins.version>2.204</jenkins.version>` i.e. without the last digit ...?

Reply:
That's ill adviced as you will usually want to use LTS version which are 3 digits.

Reply:
Usually you want to specify another jenkins.version than the one specified in the plugin parent pom

Reply:
Oh right. I had thought that by leaning on the parent POM's Jenkins version, rather than overriding it, it might offer some sort of conformance guarantee. Will set it separately